### PR TITLE
Makefile: Make Debian/RedHat package revisions settable

### LINKED ISSUE #9995: minikube_1.16.0-0_amd64.deb: package architecture (arm64) does not match system (amd64)
**Steps to reproduce the issue:** 

1. Download amd64 debian package of latest release: https://github.com/kubernetes/minikube/releases/download/v1.16.0/minikube_1.16.0-0_amd64.deb
2. Inspect the package file
3. It's marked as an arm64 package not an amd64 package
4. It has stray files in it that are clearly not meant to be there

**Full output of failed command:** 
```
$ dpkg --info minikube_1.16.0-0_amd64.deb ; dpkg --contents minikube_1.16.0-0_amd64.deb 
 new Debian package, version 2.0.
 size 20578594 bytes: control archive=389 bytes.
     381 bytes,    11 lines      control              
 Package: minikube
 Version: 1.16.0
 Section: base
 Priority: optional
 Architecture: arm64
 Maintainer: Thomas Strömberg <t+minikube@stromberg.org>
 Description: Minikube
  minikube is a tool that makes it easy to run Kubernetes locally.
  minikube runs a single-node Kubernetes cluster inside a VM on your
  laptop for users looking to try out Kubernetes or develop with it 
  day-to-day.
drwxr-xr-x root/root         0 2020-12-17 18:41 ./
drwxr-xr-x root/root         0 2020-12-17 18:41 ./minikube_deb_template/
drwxr-xr-x root/root         0 2020-12-17 18:41 ./minikube_deb_template/DEBIAN/
-rw-r--r-- root/root       416 2020-12-17 18:41 ./minikube_deb_template/DEBIAN/control
drwxr-xr-x root/root         0 2020-12-17 18:41 ./usr/
drwxr-xr-x root/root         0 2020-12-17 18:41 ./usr/bin/
-rwxr-xr-x root/root  57925077 2020-12-17 18:41 ./usr/bin/minikube
```

Note: `Architecture: arm64` and `./minikube_deb_template/...` in the contents.

However the actual binary contained within _is_ an amd64 one:
```
# extract the contents of the package
$ file minikube
minikube: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, Go BuildID=6LZF01FcZrGBprW2lwYI/gWzE5fRYw4NT_meHguiF/54zHaIwKVnjc2RZCfg6t/T0VXjXYTfvHZFrWQslv4, not stripped
```

### PR DESCRIPTION
**Pull request by tstromberg**
Used to push a package revision to help resolve #9995

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -461,12 +464,12 @@ out/docs/minikube.md: $(shell find "cmd") $(shell find "pkg/minikube/constants")
 	go run -ldflags="$(MINIKUBE_LDFLAGS)" -tags gendocs hack/help_text/gen_help_text.go
 
 deb_version:
-	@echo $(DEB_VERSION)
+	@echo $(DEB_VERSION)-$(DEB_REVISION)
```
Comment (afbjorklund) [inline Makefile:467]: This breaks my test (then again, nothing else uses "deb_version" afaik...)

But it is now different from the rpm_version, so please revert this line ?

**Review by afbjorklund — CHANGES_REQUESTED** [review]

**Comment by medyagh** [comment]
@tstromberg please take a look at the comment. 
PTAL

File: Makefile (hunk 1 above)
Comment (tstromberg) [reply Makefile:467]: which test?

The intent here is to make it so that we can st DEB_REVISION to a non-zero value if required.

Comment (afbjorklund) [reply Makefile:467]: Maybe change `rpm_version` to match, then ?

Comment (tstromberg) [reply Makefile:467]: PTAL.

**Comment by tstromberg** [comment]
Both makefile rules now behave the same:

```
⋊> ~/s/minikube on deb_revision ⨯ make rpm_version                                                                                                                                                 1.16.0-0
⋊> ~/s/minikube on deb_revision ⨯ make deb_version                                                                                                                                                 1.16.0-0
```

File: Makefile [hunk 2]
```diff
@@ -463,31 +466,33 @@ out/docs/minikube.md: $(shell find "cmd") $(shell find "pkg/minikube/constants")
...
 
 rpm_version:
-	@echo $(RPM_VERSION)
+	@echo $(RPM_VERSION)-$(RPM_REVISION)
 
-out/minikube-$(RPM_VERSION).rpm: out/minikube-$(RPM_VERSION)-0.x86_64.rpm
+out/minikube-$(RPM_VERSION).rpm: out/minikube-$(RPM_VERSION)-$(RPM_REVISION).x86_64.rpm
 	cp $< $@
 
 out/minikube-$(RPM_VERSION)-0.%.rpm: out/minikube-linux-%
```
Comment (afbjorklund) [inline Makefile:498]: Here is one more

**Review by afbjorklund — CHANGES_REQUESTED** [review]
Found one more line

**Review by afbjorklund — APPROVED** [review]

**Pull request merged**